### ISSUE TITLE
Zookeeper session lost, state: CLOSED - terminology

### ISSUE THREAD

**Issue by dabcoder** [python-zk/kazoo#562]
I've got an question (more than an issue), apologies if it has already been raised but I do not believe it's the case.

I am seeing this in my logs:

```
Zookeeper connection established, state: CONNECTED
Zookeeper session lost, state: CLOSED
```

https://github.com/python-zk/kazoo/blob/88b657a0977161f3815657878ba48f82a97a3846/kazoo/client.py#L533

And there's absolutely no issue whatsoever but I find the terminology a bit puzzling. As in the state is closed and by "session lost" I understand it as "the session has ended". So:

1) Can anyone confirm that my understanding is correct - that it is not an actual problem
2) Why is the `CLOSED` state in the same tuple as the other values `AUTH_FAILED` and `EXPIRED_SESSION`: https://github.com/python-zk/kazoo/blob/88b657a0977161f3815657878ba48f82a97a3846/kazoo/client.py#L71-L72 if closed is a "normal" behavior?

**Comment by dabcoder** [comment]
Any feedback on the above @StephenSorriaux? My understanding may not be correct, so would be interested in hearing other people thoughts on the topic. Thanks!

**Comment by StephenSorriaux** [comment]
Hello,

1. Your understanding is correct, this is a normal behavior. `lost` might not be the best wording though.

2. `LOST_STATES` are states that mean that not connection is available and that a (re-)connection is required to communicaticate with the server

You can find more information about states in the [official Zookeeper documentation](https://zookeeper.apache.org/doc/r3.5.5/zookeeperProgrammers.html#ch_zkSessions).

**Comment by dabcoder** [comment]
Thanks. For 1. I indeed think that we could use `closed` instead of `lost` in the wording.
For 2., (thanks for pointing me to the docs), _If an unrecoverable error occurs, such as session expiration or authentication failure, or if the application explicitly closes the handle, the handle will move to the CLOSED state_, so we could also update the `LOST_STATES` tuple to `CLOSED_STATES` instead?

```
CLOSED_STATES = (KeeperState.EXPIRED_SESSION, KeeperState.AUTH_FAILED,
               KeeperState.CLOSED)
```

**Comment by StephenSorriaux** [comment]
I agree, this would make more sense and follow the official documentation.
